content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -82,12 +82,11 @@ spec:
 You can define one or multiple `topologySpreadConstraint` to instruct the kube-scheduler how to place each incoming Pod in relation to the existing Pods across your cluster. The fields are:
 
 - **maxSkew** describes the degree to which Pods may be unevenly distributed.
-  It's the maximum permitted difference between the number of matching Pods in
-  any two topology domains of a given topology type. It must be greater than
-  zero. Its semantics differs according to the value of `whenUnsatisfiable`:
+  It must be greater than zero. Its semantics differs according to the value of `whenUnsatisfiable`:
   - when `whenUnsatisfiable` equals to "DoNotSchedule", `maxSkew` is the maximum
     permitted difference between the number of matching pods in the target
-    topology and the global minimum.
+    topology and the global minimum 
+    (global minimum is the minimum number of pods matching the label selector in all topology domains).
```

Comment:
this is still not reading well. Any suggestions @sftim ?

Example: you have 3 zones with 0, 2 and 3 matching pods respectively. The global minimum is 0.

Reply:
It helps to understand by adding an example right after the global minimum definition. How about we change it to
```
(global minimum is the minimum number of pods matching the label selector in all topology domains. For example, if you have 3 zones with 0, 2 and 3 matching pods respectively, The global minimum is 0.)
```

Reply:
I think we don't need to repeat "global minimum" within the parenthesis.

Other than that, I think a better phrasing is `(the minimum number of pods that match the label selector in a topology domain. For example...`